### ISSUE TITLE
Add ability to select folder with gum file

### ISSUE THREAD

**Issue by TheGuit** [charmbracelet/gum#193]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Adding an option (ie. --directory) to be able to select a folder.

**Describe alternatives you've considered**
For example using enter to navigate and space to select the folder. Or → to navigate and enter to select.

**Comment by maaslalani** [comment]
I agree this would be super useful. I think this could also mean introducing a `--file` flag.

```bash
gum file --directory # allow directories
gum file --file # allow files (defaults to true)
gum file --file=false --directory # only allows directories to be selected
```

**Comment by maaslalani** [comment]
This feature is now implemented in #211, thanks to @fedeztk! 🎉
